core/dbt/events/functions.py

```diff
@@ -50,14 +49,6 @@
 format_json = False
 invocation_id: Optional[str] = None
 
-# Colorama needs some help on windows because we're using logger.info
```

Comment:
@nathaniel-may I know you added this in PR #https://github.com/dbt-labs/dbt-core/pull/4474. I don't see a difference with it included or not in my testing. Can you think of scenario for me to test that would make code actually important here? It looks like we always call the logger.py that has the initialization we want, but I wanted to check with you first